src/components/ActivityCard.js

```diff
@@ -6,26 +6,27 @@ import { DeleteButton } from './Button.js';
 import DeleteModal from './DeleteModal.js';
 import MappedPlaceholderPictures from './MappedPlaceholderPictures.js';
 
+import useStore from '../hooks/useStore.js';
+
 import deleteIcon from '../images/binIcon.svg';
 import nextIcon from '../images/nextIcon.svg';
 
 export default function ActivityCard({
-  photo,
-  errorMessage,
   onDeleteActivity,
   nameOfSelectedFriend,
   nameOfSelectedActivity,
   nameOfSelectedCategory,
-  showBin,
-  handleResetPage,
-  id,
+  photo,
 }) {
   const [show, setShow] = useState(false);
 
+  const showBin = useStore(state => state.showBin);
+  const handleResetPage = useStore(state => state.handleResetPage);
+
   return (
     <>
       <WrapperCard>
-        {!photo > 0 ? (
+        {photo === '' ? (
```

Comment:
Why is the `photo` prop a string type? 
Since an empty string is falsy, you can do
```suggestion
        {!photo ? (
```